### PR TITLE
878 - styles(dashboard.menubar): align "share" to right, and mobile fixes

### PR DESCRIPTION
**Pull request by hiaux0**
#### After
- All actions visible
1: before
2: after
image

- Share button (case, if only button in UI)
image

- Mobile
image

### CODE REVIEW

File: src/dealDashboard/dealMenubar/dealMenubar.html [hunk 1]
```diff
@@ -10,11 +10,14 @@
...
     SHARE
   </a>
-  <a
-    if.bind="canEdit"
-    route-href="route.bind: deal.isOpenProposal ? 'editOpenProposal' : 'editPartneredDeal' ; params.bind: { id: deal.id, stageRoute: 'proposal' }">
-    <pbutton type="secondary">Edit deal</pbutton>
-  </a>
 
-  <pbutton click.delegate="cancelDeal()" if.bind="canCancel" type="tertiary">Cancel deal</pbutton>
+  <div class="dealActionButtons" if.bind="canEdit || canCancel">
```
Comment (hiaux0) [inline src/dealDashboard/dealMenubar/dealMenubar.html:14]: Split up "Share" from [ Edit+Cancel ]
by wrapping Edit+Cancel with its own `div`

When thinking about the problem, this emerged as the most fitting solution.
Edit+Cancel are variable, whereas "Share" is always present, thus makes sense to split up Edit+Cancel

File: src/dealDashboard/dealMenubar/dealMenubar.scss [hunk 2]
```diff
@@ -1,36 +1,48 @@
 @import "src/resources/elements/primeDesignSystem/variables";
 
 deal-menubar {
-  display: grid;
-  grid-template-columns: auto auto auto;
-  align-items: center;
+  display: flex;
```
Comment (hiaux0) [inline src/dealDashboard/dealMenubar/dealMenubar.scss:4]: flex for 
Share [Edit Cancel]

File: src/dealDashboard/dealMenubar/dealMenubar.scss [hunk 3]
```diff
@@ -1,36 +1,48 @@
...
   justify-content: end;
-  grid-column-gap: 10px;
-  grid-row-gap: 20px;
 
-  i.fas {
-    font-size: 20px;
+  .dealActionButtons {
+    display: grid;
```
Comment (hiaux0) [inline src/dealDashboard/dealMenubar/dealMenubar.scss:8]: grid for 
[Edit Cancel]

Note, after this fix, might make sense to not move grid here in the first place, but I didn't want to do tooo many changes

File: src/dealDashboard/dealMenubar/dealMenubar.scss [hunk 4]
```diff
@@ -1,36 +1,48 @@
...
+    grid-template-columns: auto auto;
+    align-items: center;
+    justify-content: end;
+    grid-column-gap: 10px;
+    grid-row-gap: 20px;
+
+    margin-left: 30px;
```
Comment (hiaux0) [inline src/dealDashboard/dealMenubar/dealMenubar.scss:15]: before: margin-right from the Share button.
after: margin-left from [Edit Cancel]

**Review by dkent600 — APPROVED** [review]

**Pull request merged**